### PR TITLE
Update documentation on how to dev to a custom namespace

### PR DESCRIPTION
**Pull request by chmouel**
Making it easier to do a deployment to a custom namespace, providing a script
instead of doing manual steps.

The shell script will replace properly the fields to target the custom namespace
and add the environment variable for the downward API using `kubectl set env`

Fix bug #917

- [ 🤷🏼‍♂️ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ 👌 ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [  👌 ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -185,28 +185,19 @@ kubectl create clusterrolebinding cluster-admin-binding \
...
-     and
-     [webhook](https://github.com/tektoncd/pipeline/blob/c1500fab83b09edadefb38bb8920a0c837d8f32b/config/webhook.yaml#L32)
-
-```yaml
-env:
-  - name: SYSTEM_NAMESPACE
-    valueFrom:
-      fieldRef:
-        fieldPath: metadata.namespace
+1. To install into a different namespace you can use this script :
```
Comment (bobcatfish) [inline DEVELOPMENT.md]: looks like we probably don't need the 1. anymore

Comment (chmouel) [reply DEVELOPMENT.md]: good point :)

Comment (bobcatfish) [reply DEVELOPMENT.md]: haha but also very very minor :bowing_woman: :bowing_woman: :bowing_woman:

**Pull request merged**